docs/Command-line-client.md

```diff
@@ -43,6 +43,19 @@ Syntax: `storm jar topology-jar-path class ...`
 
 Runs the main method of `class` with the specified arguments. The storm jars and configs in `~/.storm` are put on the classpath. The process is configured so that [StormSubmitter](javadocs/org/apache/storm/StormSubmitter.html) will upload the jar at `topology-jar-path` when the topology is submitted.
 
+When you want to ship other jars which is not included to application jar, you can pass them to `--jars` option with comma-separated string.
```

Comment:
It's not clear whether system jar or application jar will get loaded and may create some surprise if the uploaded jar doesn't work.  Maybe some warnings somewhere.